src/config.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I assume we'll need this same Config class in `js-threads/service-client`, do you think eventually it could be included with `js-textile` as a single point of entry?

Reply:
for posterity, i think the answer here is not. trying to keep threads from having to depend on textile.